tests/metrics/classification/test_f_beta.py

```diff
@@ -1,81 +1,76 @@
-import os
-import pytest
-import torch
-import numpy as np
-from collections import namedtuple
-
 from functools import partial
 
-from pytorch_lightning.metrics import Fbeta
+import numpy as np
+import pytest
+import torch
 from sklearn.metrics import fbeta_score
 
-from tests.metrics.utils import compute_batch, setup_ddp
-from tests.metrics.utils import NUM_BATCHES, NUM_PROCESSES, BATCH_SIZE, NUM_CLASSES, THRESHOLD
-
+from pytorch_lightning.metrics import Fbeta
 from tests.metrics.classification.utils import (
-    _binary_prob_inputs,
     _binary_inputs,
-    _multilabel_prob_inputs,
-    _multilabel_inputs,
-    _multiclass_prob_inputs,
+    _binary_prob_inputs,
     _multiclass_inputs,
-    _multidim_multiclass_prob_inputs,
+    _multiclass_prob_inputs,
     _multidim_multiclass_inputs,
+    _multidim_multiclass_prob_inputs,
+    _multilabel_inputs,
+    _multilabel_prob_inputs,
 )
+from tests.metrics.utils import BATCH_SIZE, NUM_BATCHES, NUM_CLASSES, NUM_PROCESSES, THRESHOLD, MetricTester
 
 torch.manual_seed(42)
 
 
-def _binary_prob_sk_metric(preds, target, average='micro', beta=1.):
+def _binary_prob_sk_metric(preds, target, average='micro', beta=1.0):
```

Comment:
can we rename these as `_sk_accuracy_binary_prob` so it clear which SK is used... like this, you fined multiple equal named fiction in several files and it very confusing...